.github/workflows/issues.yml

```diff
@@ -0,0 +1,31 @@
+---
+name: GH issues
+on:
+  workflow_call:
+    inputs:
+      ado_area_path:
+        description: 'ADO area path where ADO tickets should be created'
+        required: true
+        default: "2DegreesInvesting\\Test Team 1"
```

Comment:
Suggest not defining default here, and letting default be defined in called workflow.